src/mplfinance/plotting.py

```diff
@@ -366,6 +408,9 @@ def plot( data, **kwargs ):
                 if ax == ax4:
                     used_ax4 = True
 
+                if ptype == 'renko':
+                    ydata = renko_reformat_ydata(ydata, new_dates, dates)
```

Comment:
I love what you've done here, and with the volume, although not sure if it's correct to do.  

I was very tempted, before I looked at the code, to say we should put a check ear the top of `mpf.plot()` to **disallow** `volume=True` and `addplot` for `type='renko'.  

I wasn't sure whether and how we should handle volume and/or addplot.  That said, *what you have done is creative, and certainly seems to me the* ***most reasonable*** *thing to do (if we do anything).*  I don't see any real harm in doing it -- we can release the code this way and see whether the renko experts have any opinion on the matter.

Reply:
After doing a little more research I found that Renko charts aren't often designed to be used with volume, however, the way I have volume implemented seems to be the closest to correct and used on occasion. I think at least for the volume functionality, we could leave it in the off chance someone wants to see that data or someone is adamant that volume doesn't have a place in Renko charts.  

Addplot is a little different as it really skews the data when there are often multiple bricks created for the same date or many dates skipped if the user wants to pass in uniform data.  I was also tempted to simply disallow the use of addplot for type='renko', but I thought it might be useful to have the option if someone wants to plot a horizontal price line to mark the close price at the start of the data or something similar.

I'd definitely like to hear your thoughts on my opinions about these features!

Reply:
I added 'and ptype is not 'renko'` to the addplot conditional in plot().  This means when a user tries to include `addplot=somedict` when `type='renko'` the chart will simply be created without giving the user any feedback on why the data they passed into addplot wasn't displayed.  

I'm curious if raising an exception when a user passes an addplot when `type='renko` would be a good idea?

Reply:
Failing (or ignoring something) silently will typically confuse users.  I was thinking perhaps near the [top of `mpf.plot()`, maybe right after process kwargs](https://github.com/matplotlib/mplfinance/blob/master/src/mplfinance/plotting.py#L157), we could check something like:

    if config['type'] == 'renko' and config['addplot'] is not None:
        err = "`addplot` is not supported for `type='renko'`"
        raise ValueError(err)

That catches the error early and clearing informs the user as to what they need to do to correct the situation.  If they feel strongly that `addplot()` should be supported for `'renko'` then they will likely post and issue here on GitHub allowing us to discuss the matter and find out what they are really trying to accomplish.  We can then make an informed decision as to whether to support addplot, or provide another solution to their needs.

Reply:
Yes, this is what I had in mind. I'll add the check!